server/routes_test.go

```diff
@@ -35,15 +35,30 @@ func Test_Routes(t *testing.T) {
 	}
 	var tempModelFile string
 
-	createTestModel := func(t *testing.T, name string) {
-		f, err := os.CreateTemp("", "ollama-model")
+	createTestFile := func(t *testing.T, name string) string {
+		data := make([]byte, 6)
+
+		copy(data[0:4], []byte("GGUF"))
+		copy(data[4:6], []byte{0x2, 0})
+
+		f, err := os.CreateTemp("", name)
```

Comment:
You can use `t.TempDir()` which automatically get cleaned up when the test exits avoiding the need to `os.RemoveAll`

```suggestion
		f, err := os.CreateTemp(t.TempDir(), name)
```